warehouse/query-core/src/main/java/datawave/query/tables/RemoteEventQueryLogic.java

```diff
@@ -255,4 +258,13 @@ public EventBase transform(EventBase input) throws EmptyObjectException {
         
     }
     
+    @Override
+    public void setRemoteUserOperations(RemoteUserOperations userService) {
+        this.remoteUserOperatons = userService;
+    }
+    
+    @Override
+    public RemoteUserOperations getUserOperations() {
```

Comment:
Consider renaming the getter to `getRemoteUserOperations` be congruent with the setter
```suggestion
    public RemoteUserOperations getRemoteUserOperations() {
```

Reply:
sure, ok